py/entry_objects.py

```diff
@@ -0,0 +1,75 @@
+class UpcomingEvent:
+    """
+    Represents one entry of an upcoming event
```

Comment:
In this description, can we get a better description of what exactly an "upcoming event" entails?